public/app/features/admin/UserCreatePage.tsx

```diff
@@ -22,21 +15,29 @@ interface UserDTO {
 
 const createUser = async (user: UserDTO) => getBackendSrv().post('/api/admin/users', user);
 
-const UserCreatePage: React.FC<UserCreatePageProps> = ({ navModel }) => {
+const pageNav: NavModelItem = {
+  icon: 'user',
+  id: 'team-new',
+  text: 'Add new user',
```

Comment:
i would just make this `New user` tbh

```suggestion
  text: 'New user',
```